content/millennium/dstu2/scheduling/slot.md

```diff
@@ -94,6 +95,12 @@ Notes:
 
 * The search operation will only return free slots which are available to be booked. However, booked slots may still be retrieved when using the `_id` parameter.
 
+* The `_include` parameter may be provided once. Example: `_include=Slot:schedule`
+
+* The `_include` parameter may be provided with the `_id` parameter. Example: `_id=21265426-633867-3121665-0&_include=Slot:schedule`
+
+* Additional OAuth2 scopes are required when the `_include` parameter is provided. To include with `Slot:schedule`, the OAuth2 token must include the scope for Schedule.read.
```

Comment:
reword to match above given only the one value is allowed. maybe:

When `_include` is provided in a request to the closed endpoint, the OAuth2 token must include either the `user/Schedule.read` or the `patient/Schedule.read` scope in addition to a Slot scope.

Reply:
https://github.com/cerner/fhir.cerner.com/pull/256/commits/eb14ba57695d47945c5d478cd6afdf31bf97b788#diff-5af5ea7f1f259fa879893204fdd065b5R102